src/components/dashboard/FeedItems/FeedListItem.js

```diff
@@ -30,7 +30,7 @@ type FeedListItemProps = {
  * @param {object} props.styles
  * @returns {React.Node}
  */
-const FeedListItem = (props: FeedListItemProps) => {
+const FeedListItem = React.memo((props: FeedListItemProps) => {
```

Comment:
import { memo } from 'react', then just use memo(component fn)

Reply:
doesnt pass the refactoring conditions we discussed